tutorials/get-started-crossplane-kubernetes/index.mdx

```diff
@@ -0,0 +1,397 @@
+ - You have [created an API key](/en/docs/console/my-project/how-to/generate-api-key)
+ - You have [installed curl](https://curl.se/download.html)
+ - You have [installed kubectl](https://github.com/kubernetes/kops/blob/master/docs/install.md) on your local computer
+</Message>
+
+<Message type="note">
+   All commands use your current `kubeconfig` context and configuration.
+</Message>
+
+## Installing the Up command-line
+
+Run the following command to download and install the Upbound `up` command-line interface:
+
+````
+curl -sL "https://cli.upbound.io" | sh
+sudo mv up /usr/local/bin/
+````
+
+<Message type="note">
+   This tutorial uses `curl` but other command-line tools are available. You can refer to the [official Upbound documentation](https://docs.upbound.io/cli/#install-the-up-command-line) for more information.
+</Message>
+
+## Installing Upbound Universal Crossplane
+
+Open a terminal and run the following `up uxp install` command to install [Upbound Universal Crossplane (UXP)](https://docs.upbound.io/uxp/):
+
+````
+up uxp install
+UXP 1.9.0-up.3 installed
+````
+
+## Installing the provider into your Kubernetes cluster
+
+1. Run the following command to create a `.kube` directory in which you will store your `kubeconfig` file:
+
+````
+mkdir .kube
+````
+
+2. Run the following command to open the `config` file in your `.kube` directory. The file should be empty.
+
+````
+nano .kube/config
+````
+
+3. Paste the content of your Kubeconfig file and save the changes by following the information displayed on your terminal. Here is an example on a Mac Terminal screen.
+
+<Lightbox src="scaleway-kubeconfig-terminal-screen.webp" alt="" />
+
+4. Run the following command to install the provider into the Kubernetes cluster using the previously set Kubernetes configuration file.
+
+````
+cat <<EOF | kubectl apply -f -
+apiVersion: pkg.crossplane.io/v1
+kind: Provider
+metadata:
+ name: provider-scaleway
+spec:
+ package: xpkg.upbound.io/scaleway/provider-scaleway:v0.1.0
+EOF
+````
+
+You should get an output similar to this:
+
+````
+provider.pkg.crossplane.io/provider-scaleway Created
+````
+
+5. Run the following command to verify whether the provider has been properly installed.
+
+````
+kubectl get providers
+````
+
+You should get an output similar to this, with the value for `installed` returning `True`:
+
+````
+NAME                INSTALLED   HEALTHY   PACKAGE                                             AGE
+provider-scaleway   True        True   xpkg.upbound.io/scaleway/provider-scaleway:v0.1.0        11s
+````
+
+<Message type="note">
+   The procedure may take up to 5 minutes for `HEALTHY` to report `True`.
+</Message>
+
+6. If the `INSTALLED` field is empty, it means the process of downloading and installing the provider has failed. Run `kubectl describe providers` to get more information.
```

Comment:
The first part of this step is not a step, maybe you could convert the step into into a "tip" or a "note" message box